drivers/adodb-mysqli.inc.php

```diff
@@ -153,28 +171,64 @@ function _connect($argHostname = NULL,
 		}
 	}
 
-	// returns true or false
-	// How to force a persistent connection
+    /**
+     * Connect to a database with a persistent connection.
+     *
+     * @param string|null $argHostname The host to connect to.
+     * @param string|null $argUsername The username to connect as.
+     * @param string|null $argPassword The password to connect with.
+     * @param string|null $argDatabasename The name of the database to start in when connected.
+     *
+     * @return bool|null True if connected successfully, false if connection failed, or null if the mysqli extension
+     * isn't currently loaded.
+     */
 	function _pconnect($argHostname, $argUsername, $argPassword, $argDatabasename)
 	{
 		return $this->_connect($argHostname, $argUsername, $argPassword, $argDatabasename, true);
 	}
 
-	// When is this used? Close old connection first?
-	// In _connect(), check $this->forceNewConnect?
+    /**
+     * Connect to a database, whilst setting $this->forceNewConnect to true.
+     *
+     * When is this used? Close old connection first?
+     * In _connect(), check $this->forceNewConnect?
+     *
+     * @param string|null $argHostname The host to connect to.
+     * @param string|null $argUsername The username to connect as.
+     * @param string|null $argPassword The password to connect with.
+     * @param string|null $argDatabasename The name of the database to start in when connected.
+     *
+     * @return bool|null True if connected successfully, false if connection failed, or null if the mysqli extension
+     * isn't currently loaded.
+     */
 	function _nconnect($argHostname, $argUsername, $argPassword, $argDatabasename)
 	{
 		$this->forceNewConnect = true;
 		return $this->_connect($argHostname, $argUsername, $argPassword, $argDatabasename);
 	}
 
-	function IfNull( $field, $ifNull )
-	{
-		return " IFNULL($field, $ifNull) "; // if MySQL
-	}
-
-	// do not use $ADODB_COUNTRECS
-	function GetOne($sql,$inputarr=false)
+    /**
+     * Returns a stringing containing a MySQL IFNULL() function call for the provided values.
+     *
+     * @param mixed $field The value/column to check for NULL.
+     * @param mixed $ifNull The value/column to use if the first value/column is NULL.
+     *
+     * @return string
+     */
+    function IfNull($field, $ifNull)
+    {
+        return " IFNULL($field, $ifNull) ";
+    }
+
+    /**
+     * Do not use $ADODB_COUNTRECS.
```

Comment:
I know you just used whatever was there, but this comment is very confusing. Might as well remove it.